src/app.js

```diff
@@ -43,22 +43,22 @@ openevent.controller('AppController',
 
     app.$storage = $sessionStorage;
     if ( app.$storage.event === null ||
-        typeof(app.$storage.event) == 'undefined')
+        typeof(app.$storage.event) == "undefined")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/triplez/open-event-webapp/file/2411866147/issues/source?bid=3233857&fileBranchId=3235518#l46)